Packs/AbuseDB/Integrations/AbuseDB/AbuseDB.yml

```diff
@@ -113,8 +113,11 @@ script:
     - contextPath: IP.Address
       description: The address of the IP.
       type: String
-    - contextPath: IP.Geo.Country
-      description: The country in which the IP address is located.
```

Comment:
Removing an output can be a breaking backward compatibility change.
Lets leave it as is, and also add the other data you added